src/util/misc.js

```diff
@@ -190,19 +192,33 @@
     getBisector: function (A, B, C) {
       var AB = fabric.util.createVector(A, B), AC = fabric.util.createVector(A, C);
       var alpha = fabric.util.calcAngleBetweenVectors(AB, AC);
-      //  check if alpha is relative to AB->BC
-      var ro = fabric.util.calcAngleBetweenVectors(fabric.util.rotateVector(AB, alpha), AC);
-      var phi = alpha * (ro === 0 ? 1 : -1) / 2;
+      var phi = alpha / 2;
       return {
         vector: fabric.util.getHatVector(fabric.util.rotateVector(AB, phi)),
-        angle: alpha
+        angle: Math.abs(alpha)
       };
     },
 
     /**
-     * Project stroke width on points returning 2 projections for each point as follows:
+     * @static
+     * @memberOf fabric.util
+     * @param {Point} vector 
+     * @param {Boolean} counterClockwise the direction of the orthogonal vector 
+     * @returns {Point} the unit orthogonal vector
+     */
+    getOrthogonalUnitVector: function(vector, counterClockwise = true) {
```

Comment:
`counterClockwise = true` will probably cause lint to go mad cause fabric's js is so old.
Fortunately, the next PR to enter fabric is TS so that won't be a problem.
But I do intent on merging this PR before that.
We could negate the arg to `clockwise` - but let's leave as is and TS will fix it. We can live with that.

Reply:
Rethinking, we can use `fabric.util.rotateVector` instead. Better I think.

Reply:
I also thought of using the `rotateVector` function. But since the orthogonal vector is just swapping the coordinates, it seemed to me to be less prone to accuracy issues. 

I made a commit removing the optional parameter https://github.com/fabricjs/fabric.js/pull/8083/commits/b554b6d64de1529ef31f095ea5c3d7d2c254fa03

However, if you think it's better to use the `rotateVector` function, I'll refactor.